tools/workspace-plugin/src/generators/vrt-generator/schema.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+/**
+ * Create visual-regression package within FluentUI v9 component package
+ */
+export interface VisualRegressionSchema {
+  /**
+   * The name of the component package
```

Comment:
bad copy paste ?

how is this supposed to work ? user provides v9 project ( react-card ) for instance and we create the new visual-regression project ?

Reply:
Yes, I thought it to work in the similar way as we have other generators, to have a prompt with dropdown of projects. Is there a better way?